src/stylesheets/components/_article.scss

```diff
@@ -8,8 +8,18 @@
 article {
   margin: $base-spacing-unit-extra-large 0 $base-spacing-unit-small;
 
+  > * {
```

Comment:
Can we move this below the `@include media($desktop)` line? > * is for child elements, but the mixin affects the `article` element